test/integration/wide_angle_camera.cc

```diff
@@ -227,28 +198,13 @@ void WideAngleCameraTest::WideAngleCamera(
 
   // Clean up
   engine->DestroyScene(scene);
-  gz::rendering::unloadEngine(engine->Name());
```

Comment:
These are removed because they are handled by the test fixture now: 

https://github.com/gazebosim/gz-rendering/pull/685/files#diff-9b5026daaee2123f95b0ee0a29787297a9d106b248731492d1c15796940d0fa0R85-R89

This also guarantees that any resources allocated in the test will be deallocated before the test is torn down.